cfg/githubfieldmapper.go

```diff
@@ -0,0 +1,235 @@
+package cfg
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/andygrunwald/go-jira"
+	"github.com/coreos/issue-sync/lib/models"
+	"strings"
+	"time"
+)
+
+type FieldMapper interface {
+	MapFields(issue *models.ExtendedGithubIssue) jira.IssueFields
```

Comment:
I wonder if we could find a clearer name for this. Mapping from what to what?

Reply:
Since the data flow is one-way I figured this should be self-explanatory but I can call it githubToJiraFieldMapper.

Reply:
Going a step farther, does it map a single field to a single field, or a group to a group? Does it just map field ids, or something more elaborate? I'm not certain a better name is needed, for either the interface or the method, but I don't yet feel like I have a strong understanding of the details.